### PR TITLE
Add pull request statuses menu samples to contributions-guide

### PR DESCRIPTION
**Pull request by rsemenov**
Added samples of how to extend pull request status menu.

image

### CODE REVIEW

**Comment by rsemenov** [comment]
@trevorsg could you take a look at this?

File: contributions-guide/vss-extension.json [hunk 1]
```diff
@@ -665,6 +665,50 @@
...
+                "statusGenre": "fabricam-ci-service",
+                "statusName": "ci-build"
+            }
+        },
+        {
+            "id": "statusMenuItem_1",
+            "type": "ms.vss-web.action",
+            "description": "Executes custom action for the status",
+            "targets": [
+                "ms-samples.samples-contributions-guide.sample-pr-status-menu"
```
Comment (trevorsg) [inline contributions-guide/vss-extension.json]: Can you use a relative target here? e.g. `"targets": [ ".sample-pr-status-menu" ]` This extension is sometimes used as a starting point for extension developers, so they may change the publisher. Using the relative target ID, it doesn't matter what the publisher or extension name is.

File: contributions-guide/scripts/main.js [hunk 2]
```diff
@@ -29,4 +29,14 @@ var showPropertiesMenuProvider = (function () {
 
 VSS.register("showProperties", function (context) {
     return showPropertiesMenuProvider;
+});
+
+VSS.register("statusCustomAction", function (context) {
+    return {
+        execute: function (context) {
+            let statusName = context.status.context.genre + "/" + context.status.context.name;
```
Comment (trevorsg) [inline contributions-guide/scripts/main.js]: Use var - let won't work on IE11.

File: contributions-guide/vss-extension.json (hunk 1 above)
Comment (rsemenov) [reply contributions-guide/vss-extension.json]: It makes sense. Updated to use relative targets.

File: contributions-guide/scripts/main.js (hunk 2 above)
Comment (rsemenov) [reply contributions-guide/scripts/main.js]: Done. Thanks!

**Pull request merged**